includes/application.php

```diff
@@ -46,7 +46,7 @@ function ($class_name) {
 
     // Validate session token, if exists
     if (!empty($_SESSION['session_token'])) {
-        $UserAuth = FirstAide\Authentication::withSessionToken($DB, $_SESSION['session_token']);
+        $UserAuth = FirstAide\Authentication::createInstanceWithSessionToken($DB, $_SESSION['session_token']);
```

Comment:
We can import the namespace 'FirstAide' in this file rather than using FirstAide\ in all the places